src/server/config/auth.js

```diff
@@ -0,0 +1,19 @@
+export const superToken = 'yodaROX';
+
+export function validiateFB(accessToken) {
+  if (accessToken == superToken) {
```

Comment:
This method will validiate to facebook when signin access_token is right. It will add soon.